Makefile

```diff
@@ -21,7 +21,8 @@ test-assignment:
 	@cp $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE) $(OUTDIR)/$(TESTFILE)
 	@sed 's/pending/it/g' $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) > $(OUTDIR)/$(SPECFILE)
 	@cd $(OUTDIR) && crystal spec $(SPECFILE)
-	@echo "\n"
+	crystal tool format --check $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE)
```

Comment:
I think you can just do `crystal tool format --check $(EXERCISESDIR)/$(ASSIGNMENT)` and it will do both files in the directory. Thoughts?

```shell
Maxs-MacBook-Pro:exercism_xcrystal max$ crystal tool format --check exercises/hamming
Error: formatting 'exercises/hamming/example.cr' produced changes
Error: formatting 'exercises/hamming/hamming_spec.cr' produced changes
```